templates/openstack/install.html

```diff
@@ -103,7 +99,7 @@ <h2>Single-node OpenStack deployment</h2>
   </div>
   <div class="row">
     <div class="col-8">
-      <p class="u-sv3">These instructions use <a href="https://snapcraft.io/microstack" class="p-link--external">MicroStack</a>, an upstream single-node OpenStack deployment which can run directly on your workstation. MicroStack is OpenStack in a <a href="https://snapcraft.io/" class="p-link--external">snap</a> which means that all services and supporting libraries are together in a single package that can be easily installed, upgraded or removed. MicroStack includes all key OpenStack components: Keystone, Nova, Neutron, Glance, and is evolving extremely fast. You can use it for development, prototyping and testing, but it is also perfectly suitable for the network edge, IoT and appliances.</p>
+      <p class="u-sv3">These instructions use <a href="https://snapcraft.io/microstack" class="p-link--external">MicroStack</a> &dash; OpenStack in a snap, a pure upstream OpenStack distribution designed for small scale and edge deployments, that which can be installed and maintained with a minimal effort.</p>
```

Comment:
```suggestion
      <p class="u-sv3">These instructions use <a href="https://snapcraft.io/microstack" class="p-link--external">MicroStack</a> &dash; OpenStack in a snap. MicroStack is a pure upstream OpenStack distribution, designed for small scale and edge deployments, that can be installed and maintained with a minimal effort.</p>
```
Doesn't match copydoc